linera-explorer/src/lib.rs

```diff
@@ -567,13 +567,11 @@ async fn page(
         }
         "blocks" => blocks(node, chain_id, None, Some(20)).await,
         "applications" => applications(node, chain_id).await,
-        "application" => match find_arg(args, "app").map(|v| parse(&v)) {
-            None => Err(anyhow!("unknown application")),
-            Some(app_js) => {
-                let app = from_value::<Application>(app_js).unwrap();
-                application(app).await
-            }
-        },
+        "application" => {
+            let app_arg = find_arg(args, "app").context("unknown application")?;
+            let app = from_value::<Application>(parse(&app_arg)).unwrap();
```

Comment:
Why the `unwrap` here?

Reply:
I didn't add that.
But you're right, we could probably return an error instead.

Reply:
Ironically, I would actually have to use `map_err(|error| anyhow!(…))?` here!
But all the other places in this file where `from_value` is called use `expect`, so I'd rather change the `unwrap` to an `expect` for now, for consistency.